src/translations/translations_EN.json

```diff
@@ -37,6 +37,10 @@
     "heading": "Privacy policy"
   },
   "resource": {
+    "access_types": {
+      "open": "Open",
+      "private": "private"
```

Comment:
stor P

Reply:
access_typen "private" er forvøvrig det som benyttes av gamle-dlr når man setter "begrenset" deling. Skulle jeg ha endret oversettelsen til "Restricted"? Var jo også snakk om å innføre en private access_type.

Reply:
Skjønte ikke helt. Dette er oversettelsen, ikke typen, ja ?

Reply:
Det er oversettelsen ja. Tenkte mer på når vi skal ta i bruk typen senere. 
Skrev opp "private" siden jeg allikevel hadde vært på gamle dlr og sett a variabel navnet for "begrenset" ble sendt over til backend som "private"